frontend/packages/console-plugin-sdk/src/typings/extension.ts

```diff
@@ -27,7 +27,7 @@ export type Extension<P> = {
  *
  * ```json
  *  {
- *    "name": "@console/demo-plugin",
+ *    "name": "@console/console-demo-plugin",
```

Comment:
This change shouldn't be necessary (but I still like you didn't forget about it).